### PR TITLE
Dom/validate vg schema

### PR DESCRIPTION
**Pull request by domoritz**

### CODE REVIEW

**Comment by domoritz** [comment]
Fixes #613

File: editor/validate.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+  if (!valid) {
+    console.error(validator.getLastErrors());
+  }
+};
+
+var validateVg = function(vgspec) {
+  var validator = new ZSchema();
+
+  var cb = function() {
```
Comment (kanitw) [inline editor/validate.js]: `callback`?

**Comment by kanitw** [comment]
This is a step toward #613, but I think we add a unit test that loads all the examples and verify if it produces sensible vega spec too.  Therefore, whenever we break the output a test is failed -- rather than having to rely on obscure `console.error` in the gallery.

**Comment by kanitw** [comment]
I'll merge this for now as it's still useful.

**Pull request merged**